ubuntu-18.04/mysql.sh

```diff
@@ -13,6 +13,56 @@ readonly MYSQL_USER_OPTIONS_FILE_PATH="${HOME}/.my.cnf"
 
 # Functions
 
+#######################################
+# Create a MySQL database and crendentials from passed arguments.
+# The database and credentials are created if not existing.
+# The new database name is set to the credential username.
+# Arguments:
+#   The database server FQDN, a string.
+#   The database adminstrator's username, a string.
+#   The database adminstrator's password, a string.
+#   The new database credentials username, a string.
+#   The new database credentials password, a string.
+# Outputs:
+#   Writes message to STDOUT.
+#######################################
+function create_database_and_credentials() {
+
+  # Parameters
+  local database_server_fqdn="$1"
+  local database_server_admin_username="$2"
+  local database_server_admin_password="$3"
+  local database_server_new_credentials_username="$4"
+  local database_server_new_credentials_password="$5"
+
+  # Variables
+  local database_server_new_credentials_database="${database_server_new_credentials_username}"
+
+  mysql::create_user_options_file \
+    "${database_server_admin_username}" \
+    "${database_server_admin_password}" \
+    "${database_server_fqdn}" \
+    "5432" \
+    "postgres"
```

Comment:
postgres?

Reply:
"The postgres database is a default database meant for use by users, utilities and third party applications."
Ref.: https://www.postgresql.org/docs/current/app-initdb.html

Reply:
Good catch...